src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -161,6 +191,12 @@ private void UpdateStateMachine()
                     // the extension, then reenables the extension. We will show the gold bar after the switch
                     // if there is still a pending show.
 
+                    // If ReSharper was suspended and the user closed and reopened VS, we want to reset the gold bar
+                    if (currentStatus == ReSharperStatus.Suspended && _isFirstRun)
```

Comment:
shouldn't it be != ResharperStatus.Enabled to show infobar for uninstalled case?

Reply:
Yes, I changed it.